setup/steps/check_pgss_available.go

```diff
@@ -0,0 +1,30 @@
+package steps
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/pganalyze/collector/setup/query"
+	s "github.com/pganalyze/collector/setup/state"
+)
+
+var CheckPgssAvailable = &s.Step{
+	Description: "Prepare for pg_stat_statements install",
```

Comment:
We should update this description (since we don't actually install anything here, we just check things)